<ui version="4.0" >
 <class>pqPythonToolsWidget</class>
 <widget class="QWidget" name="pqPythonToolsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Python Tools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>335</width>
        <height>323</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget" >
         <property name="toolTip" >
          <string/>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="DirectoryViewTab" >
          <property name="toolTip" >
           <string>This tab displays a directory view with python scripts existing on disk.</string>
          </property>
          <attribute name="title" >
           <string>Directory View</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout" >
           <item row="0" column="0" colspan="2" >
            <layout class="QHBoxLayout" name="horizontalLayout" >
             <item>
              <widget class="QLineEdit" name="ScriptDirectoryEntry" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Enter path to script directory</string>
               </property>
               <property name="whatsThis" >
                <string>This text entry holds the path to the currently displayed script directory.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ChooseDirectoryButton" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize" >
                <size>
                 <width>27</width>
                 <height>27</height>
                </size>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Choose script directory</string>
               </property>
               <property name="text" >
                <string>...</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" >
            <widget class="QTreeView" name="ScriptView" />
           </item>
           <item row="1" column="1" >
            <layout class="QVBoxLayout" name="verticalLayout_3" >
             <item>
              <widget class="QPushButton" name="RefreshButton" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Refresh the directory view</string>
               </property>
               <property name="text" >
                <string>Refresh</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="NewScriptButton" >
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Open a new script file in the editor</string>
               </property>
               <property name="text" >
                <string>New script</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="RunSelectedButton" >
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Run the script that is currently selected in the directory view</string>
               </property>
               <property name="text" >
                <string>Run selected</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="AddToMacrosButton" >
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Add the currently selected script to the macros set</string>
               </property>
               <property name="text" >
                <string>Add to macros</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer" >
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="MacrosTab" >
          <property name="toolTip" >
           <string>This tab manages the current set of python macros</string>
          </property>
          <attribute name="title" >
           <string>Macros</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <item>
            <widget class="QListWidget" name="MacroListBox" />
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5" >
             <item>
              <widget class="QPushButton" name="RemoveMacroButton" >
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Remove the selected macro</string>
               </property>
               <property name="text" >
                <string>Remove</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="showMenuCheck" >
               <property name="toolTip" >
                <string>Hide or show the macros menu in the main menu bar</string>
               </property>
               <property name="text" >
                <string>Show menu</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2" >
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>204</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="TraceTab" >
          <property name="toolTip" >
           <string>This tab contains buttons for controlling python trace</string>
          </property>
          <attribute name="title" >
           <string>Trace</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3" >
           <item row="0" column="0" colspan="3" >
            <spacer name="verticalSpacer_3" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>80</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" >
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>99</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" >
            <layout class="QVBoxLayout" name="verticalLayout_4" >
             <item>
              <widget class="QPushButton" name="StartTraceButton" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="focusPolicy" >
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="toolTip" >
                <string>Start trace.  Clears the current trace and restarts it if trace is already active.</string>
               </property>
               <property name="text" >
                <string>Start Trace</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="StopTraceButton" >
               <property name="toolTip" >
                <string>Stops active trace.  The trace output generated so far will still be available to show, edit, and save.</string>
               </property>
               <property name="text" >
                <string>Stop Trace</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ShowTraceButton" >
               <property name="toolTip" >
                <string>Prints the current trace output in the python shell for display</string>
               </property>
               <property name="text" >
                <string>Show Trace</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="EditTraceButton" >
               <property name="toolTip" >
                <string>Opens the current trace output in the script editor window</string>
               </property>
               <property name="text" >
                <string>Edit Trace</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="SaveTraceButton" >
               <property name="toolTip" >
                <string>Prompts for a filename and saves the current trace output to disk</string>
               </property>
               <property name="text" >
                <string>Save Trace</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="2" >
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>98</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="3" >
            <spacer name="verticalSpacer_4" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>80</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
